### PR TITLE
Feature/76/78

### LINKED ISSUE #76: Postmark could be moved as a production gem
`gem 'postmark-rails' ` is an external paid service. It would make more sense to put it in the `:production` bundler group. 

It shouldn't be required for development.

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**
Resolves #76 #78 
Ping @Mr0grog

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-  password = SecureRandom.uuid
+  password = 'PASSWORD'
   user.email = 'seed-admin@example.com'
   user.password = password
   user.confirmed_at = Time.now
 
   edit_path = Rails.application.routes.url_helpers.edit_user_registration_path
   puts "\n\n------------------------------------------------------------"
   puts "Admin user created with e-mail: #{user.email} and password: #{password}"
-  puts "Point your browser to http://localhost:3000#{edit_path} to change it."
```
Comment (Mr0grog) [inline db/seeds.rb]: I wonder if this message would still be useful.

Regardless, please either put this back or remove the line declaring `edit_path` above to resolve the Rubocop error.

**Comment by Mr0grog** [comment]
One more thing: you should add yourself to the [contributors section of the README](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/README.md#contributors)

**Comment by KrzysztofMadejski** [comment]
Thanks for welcoming me there. I've added a line in #79 not to run into README conflicts.

**Comment by Mr0grog** [comment]
Thanks for showing up and helping out! 🎉

**Pull request merged**